recipes/recipes/lfortran/recipe.yaml

```diff
@@ -0,0 +1,38 @@
+context:
+  name: lfortan
+  version: "0.35.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+# - url: https://github.com/lfortran/lfortran/archive/refs/tags/v${{ version }}.tar.gz
+#   sha256: 2e54c5cfe9a94fc572bfc39c51496203636f78fb04e8bc0809577c4e1713a2ac
+  - git: https://github.com/lfortran/lfortran.git
+    rev: 0ea994d2c6bf834ca73fd298bd5e4256db4769b4
+
+build:
+  number: 1
+
+requirements:
+  build:
+  - ${{ compiler('cxx') }}
+  - python
+  - re2c
+  - bison ==3.4
+  - llvmdev ==11.0.1
```

Comment:
It builds with llvmdev v15 also. But after that, newer versions moved header files around and it breaks.